### PR TITLE
Fix RTL styling for Media Text block

### LINKED ISSUE #17922: Media/Text Block: RTL styles for media alignment not working
**Describe the bug**

The media/image block does not respect media alignments.

**To reproduce**

1. Go to `/wp-admin/options-general.php` and change the site language to an RTL language, e.g. Hebrew
2. Go to `/wp-admin/post-new.php`
3. Add the following two media/image blocks:
3.1 Media: left / Text: right
3.2 Media: right / Text: left

**Expected behavior**

- Left-aligned images appear on the left and right-aligned images appear on the right.

**Screenshots**

- Left-aligned images appear on the right and right-aligned images appear on the left.

**Desktop (please complete the following information):**

 - OS: iOS 10.14.6 (18G103)
 - Browser Chrome Version 77.0.3865.90 (Official Build) (64-bit)

**Additional context**

- WordPress 5.2.3 (no Gutenberg plugin active)
- This issue was discovered in https://github.com/WordPress/twentytwenty/issues/773#issuecomment-541302165

### PR DESCRIPTION
**Pull request by richtabor**
## Description
This PR closes #17922 by adding proper `!rtl` comments to ignore the auto-generating rtl styling that interfered with the Media & Text block's `mediaPosition` control (where folks decide to set media to display on the left, or right). 

## How has this been tested?
Tested on supported browsers.

## Screenshots 

**LTR front of site:**
fos

**RTL front of site:**
rtl-fos

**LTR editor:**
editor

**RTL editor:**
rtl-editor

## Types of changes
SCSS change only

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate.

### CODE REVIEW

File: packages/block-library/src/media-text/style.scss [hunk 1]
```diff
@@ -1,7 +1,12 @@
 .wp-block-media-text {
+	// This block's direction should not be manipulated by rtl, as the mediaPosition control does.
+	/*!rtl:begin:ignore*/
+	direction: ltr;
```
Comment (jorgefilipecosta) [inline packages/block-library/src/media-text/style.scss]: Unfortunately, this makes it impossible to have rtl paragraphs nested inside the media text.
Should we add direction ltr (not ignored) to the container of the nested blocks?

**Review by jorgefilipecosta — COMMENTED** [review]
Hi @richtabor, thank you for submitting a fix for this issue 👍

File: packages/block-library/src/media-text/style.scss (hunk 1 above)
Comment (richtabor) [reply packages/block-library/src/media-text/style.scss]: Ah you're right. It seems to work on the front of site, but not within the editor. You mean the actual `.wp-block-media-text__content` container, correct?

Comment (richtabor) [reply packages/block-library/src/media-text/style.scss]: I've just pushed an update which adds `direction: ltr` to the `.wp-block-media-text__content` container, ensuring the content containers still abide by proper rtl support. 

Here's a screenshot with a rtl language applied:
Screen Shot 2019-12-03 at 8 44 13 AM

**Review by karmatosed — APPROVED** [review]
In testing, this works for me so going to merge this, thanks!

**Pull request merged**